util/random.go

```diff
@@ -29,3 +31,30 @@ func RandomUint64() (uint64, error) {
 	}
 	return binary.LittleEndian.Uint64(k), nil
 }
+
+// Generates a 64 character hex encoded string has a prefix not equal to any
+// of the existingPrefixes
+func RandomUniqueToken(existingPrefixes []string, tokenSize int) (string, error) {
```

Comment:
Why is this in util?  If tokens are generated in politeiad, wouldn't it make more sense to have this as a politeiad method?

Reply:
Moved it.